engine/src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java

```diff
@@ -202,6 +202,16 @@ public void doHttpRequest(Event event, Endpoint endpoint, HttpRequest<Buffer> re
 
                 boolean serverError = false;
                 boolean shouldResetEndpointServerErrors = false;
+                if (isEmailEndpoint) {
+                    JsonObject details = new JsonObject();
+                    try {
+                        int totalRecipients = payload.getJsonArray("emails").getJsonObject(0).getJsonArray("bccList").size();
+                        details.put("total_recipients", totalRecipients);
+                        history.setDetails(details.getMap());
+                    } catch (Exception ex) {
+                        Log.info("This email payload doesn't use bccList");
```

Comment:
This would be an issue the dev team has to investigate, let's raise the log level. Since the `catch` block could be executed for other reasons than the lack of `bccList` field, we should also include the exception in the log entry.
```suggestion
                        Log.error("Could not set the total_recipients field in the history details", e);
```

Reply:
Ok